add-on/src/popup/browser-action/context-actions.js

```diff
@@ -45,6 +45,7 @@ export function contextActions ({
   onFilesCpImport
 }) {
   const activeCidResolver = active && isIpfsOnline && isApiAvailable && currentTabCid
+  // TODO(whizzzkid): Remove this once we have a better way to handle this.
   const activeFilesCpImport = active && isIpfsOnline && isApiAvailable && !ipfsNodeType.startsWith('embedded')
```

Comment:
Do we want to remove `activeFilesCpImport` entirely? or just the `embedded` check?